### DISCUSSION TITLE
Tasmota Zählerdaten in OpenDTU anzeigen lassen

### DISCUSSION THREAD

**Discussion by Forstgrund** [tbnobody/OpenDTU#1683] [Ideas]
Hat sich schon mal jemand damit beschäftigt, die Daten vom Tasmota Zählerstand in der OpenDTU anzeigen zu lassen?
Also nur die beiden Dinge wie Zählerstand und Momentanverbrauch. Das wäre doch mal ne coole Sache..

**Comment by ituri** [comment]
[OpenDTU on Battery](https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Live-View) macht sowas.

**Reply by Forstgrund** [reply]
Ja das sieht gut aus....habs mal installiert. Jetzt muss ich erst mal sehen wie ich die Stromzähler Daten da eintragen muss. Dachte die IP reicht aus...aber er will noch einiges dazu haben
